lib/base/error-handler.js

```diff
@@ -80,6 +80,7 @@ const log = require('npmlog'),
             "INVALID_OBJECT" : "Object format error",
             "INVALID_MODE" : "Please specify an option, either '--add' or '--modify'",
             "INVALID_CAPTURE_FORMAT" : "Please specify the file extension(.png, .bmp or .jpg)",
+            "INVALID_CSV_FORMAT" : "Please specify the file extension(.csv)",
```

Comment:
I think the message should be different because it does not pick one out of several possible, but only supports one specific one.

"Please change the file extension to csv"
Or "The file extension only supports csv"
seems to be good.

Reply:
Done.